ui/components/component-library/button-base/README.mdx

```diff
@@ -65,6 +65,8 @@ import { ButtonBase } from '../ui/component-library';
 
 Use the `as` box prop to change the element of `ButtonBase`. Defaults to `button`.
 
+When an `href` prop is passed it will change the element to an anchor(`a`) tag.
```

Comment:
I feel like this could be it's own story and section as well? Seems a little tucked away here?